src/errors.ts

```diff
@@ -41,5 +41,6 @@ export class MetaMaskConnectionError extends ConnectionError {
       action: t`Reload`,
       message: t`'A Metamask error caused your wallet to disconnect. Reload the page to reconnect.'`,
     })
+    this.name = 'MetaMask Connection Error'
```

Comment:
I think this should reuse the name of `ConnectionError` - the details about MM are included in the `message`.
I think this should be backed out.